icu4j/maven-build/maven-icu4j/pom.xml

```diff
@@ -254,7 +254,7 @@
                   <configuration>
                       <executable>cp</executable>
                       <arguments>
-                        <argument>../../../icu4c/LICENSE</argument>
+                        <argument>../../main/shared/licenses/LICENSE</argument>
```

Comment:
why point to the symlink, not to the actual file in root?

Reply:
> why point to the symlink, not to the actual file in root?

Ooh! This one. 

Because you might be working from an icu4j sources jar instead of out of the whole git repo.

Reply:
Makes sense. Could you please add a comment saying so, to prevent the next person from asking, or from changing it?